classes/class-wc-connect-nux.php

```diff
@@ -83,5 +97,300 @@ public function register_add_service_to_zone_pointer( $pointers ) {
+					break;
+			}
+
+			$default_content = array(
+				'title'           => __( 'Connect your store to activate WooCommerce Shipping', 'woocommerce-services' ),
+				'description'     => __( "WooCommerce Shipping is almost ready to go! Once you connect your store you'll be able to access discounted rates and printing services for USPS and Canada Post from your dashboard (fewer trips to the post office, winning).", 'woocommerce-services' ),
+				'button_text'     => $button_text,
+				'image_url'       => $image_url,
+				'should_show_jp'  => true,
+			);
+
+			$base_location = wc_get_base_location();
+			$country = isset( $base_location['country'] )
+				? $base_location['country']
+				: '';
+			switch ( $country ) {
+				case 'CA':
+					$localized_content = array(
+						'description'     => __( "WooCommerce Shipping is almost ready to go! Once you connect your store you'll be able to show your customers live shipping rates when they check out.", 'woocommerce-services' ),
+					);
+					break;
+			}
+
+			$this->show_nux_banner( wp_parse_args( $localized_content = null, $default_content ) );
+		}
+
+		public function show_banner_after_connection() {
+			if ( ! $this->should_display_nux_notice_on_screen( get_current_screen() ) ) {
+				return;
+			}
+
+			// Did the user just dismiss?
+			if ( isset( $_GET['wcs-nux-notice'] ) && 'dismiss' === $_GET['wcs-nux-notice'] ) {
+				WC_Connect_Options::update_option( self::SUCCESS_BANNER_IS_DISMISSED, true );
+				wp_safe_redirect( remove_query_arg( 'wcs-nux-notice' ) );
+			}
+
+			$this->show_nux_banner( array(
+				'title'          => __( 'Setup complete! You can now access discounted shipping rates and printing services' ),
+				'description'    => __( 'When you’re ready, you can purchase discounted labels from USPS, and print USPS labels at home.', 'woocommerce-services' ),
+				'button_text'    => __( 'Got it, thanks!', 'woocommerce-services' ),
+				'button_link'    => add_query_arg( array(
+					'wcs-nux-notice' => 'dismiss',
+				) ),
+				'image_url'      => plugins_url(
+					'images/nux-printer-laptop-illustration.png', dirname( __FILE__ )
+				),
+				'should_show_jp' => false,
+			) );
+		}
+
+		public function show_nux_banner( $content ) {
+			?>
+			<div class="notice wcs-nux__notice">
+				<div class="wcs-nux__notice-logo">
+					<img src="<?php echo esc_url( $content['image_url'] );  ?>">
+				</div>
+				<div class="wcs-nux__notice-content">
+					<h1><?php echo esc_html( $content['title'] ); ?></h1>
+					<p class="wcs-nux__notice-content-text">
+						<?php echo esc_html( $content['description'] ); ?>
+					</p>
+					<?php if ( $content['should_show_jp'] ) : ?>
+						<p><?php printf( esc_html__( 'By clicking "%1$s", you agree to the %2$sTerms of Service%3$s and understand that some of your data will be passed to external servers. You can find more information about how your data is handled %4$shere%5$s.', 'woocommerce-services' ),
```

Comment:
Updated with new syntax based on advice from @hewsut & added a translators comment in 6f44247d94b672d8100f6d5d1134b2fcd2a3e60f